pkg/backup/util/util.go

```diff
@@ -638,3 +663,17 @@ func canSkipSetGCLifeTime(image string) bool {
 	}
 	return true
 }
+
+// canOpenLogBackup returns if tikv supports log backup
+func canOpenLogBackup(image string) bool {
```

Comment:
rename to IsLogBackSupport, param "image" rename to tikvImage

Reply:
ok